ParabolInc/parabol
# Added updateOAuthRefreshTokens mutation

Fixes #5510. 
Cron job PR: https://github.com/ParabolInc/action-chronos/pull/6 

Tests:
1. Integrate Atlassian for a few users
2. Change the `updatedAt` to be more than 14 days ago
3. Open http://localhost:3000/admin/graphql 
4. Call the mutation below
```
mutation {
  updateOAuthRefreshTokens
}
```
5. Check if the integer output equals the number of rows which supposed to be updated

AC:
- [x] `updateOAuthRefreshTokens` exists in internal schema
- [ ] `updateOAuthRefreshTokens` when called updates all the AtlassianAuth rows older than 14 days
- [x] `updateOAuthRefreshTokens` can be called only by admins
- [ ] `updateOAuthRefreshTokens` is called 2 times per month by cron job
- [ ] Atlassian integration still works correctly
